userspace/libscap/scap.h

```diff
@@ -204,6 +204,7 @@ typedef struct scap_threadinfo
 	uint64_t pid; ///< The id of the process containing this thread. In single thread processes, this is equal to tid.
 	uint64_t ptid; ///< The id of the thread that created this thread.
 	uint64_t sid; ///< The session id of the process containing this thread.
+	uint64_t pgrp; ///< The process group of this thread.
```

Comment:
Should we call it `pgrp` or `pgid`? it looks it's called pgid on /proc/pid/status

Reply:
It's a bit confusing, it seems that in some places it been listed as `pgid` and in some `pgrp` (like the in the getter function `task_pgrp_nr`).
If you think that we need to change it I will.

Reply:
@luca3m I will change the field name to `pgid`